src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        int number = new Random().nextInt(100);
```

Comment:
let's create a constant for 100 
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names